server/build.gradle

```diff
@@ -107,7 +107,7 @@ dependencies {
     implementation('com.auth0:java-jwt:3.4.0')
 
     //local redis
-    implementation('com.github.kstyrc:embedded-redis:0.6')
+    implementation('com.github.codemonstur:embedded-redis:1.4.3')
```

Comment:
Are these changes backward compatible?

Reply:
Maybe not, but no service relies on the [Redis](https://github.com/kstyrc/embedded-redis/blob/master/src/main/java/redis/embedded/Redis.java) interface,

- The old embedded [Redis](https://github.com/kstyrc/embedded-redis/blob/master/src/main/java/redis/embedded/Redis.java) server version is [2.8.9](https://github.com/kstyrc/embedded-redis/tree/master/src/main/resources)
- The new embedded [Redis](https://github.com/codemonstur/embedded-redis/blob/master/src/main/java/redis/embedded/Redis.java) server version is [6.2.7](https://github.com/codemonstur/embedded-redis/tree/master/src/main/resources)

Due to the new embedded Redis server's dependency on `libssl.so.3`, the base image needed to be [upgraded](https://github.com/line/line-fido2-server/pull/53/commits/126d69f219b3da599c3032701556a0c14e8c586d) for backward compatibility.